src/docket/worker.py

```diff
@@ -39,6 +45,7 @@ async def __aenter__(self) -> Self:
                 await redis.xgroup_create(
                     groupname=self.consumer_group_name,
                     name=self.docket.stream_key,
+                    id="0-0",
```

Comment:
had to look this up

Reply:
Yeah I keep waffling on exactly how this is supposed to work.  Need to set up an integration test suite that can run many workers for "real" to make sure I have all this redis stuff right